app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    splitted_command = command.split()
+    if "/" not in splitted_command[-1]:
```

Comment:
Use os.path.split checking for "/" is not good practice, because it does not work on all OS